### PR TITLE
Warn about findings from rpm_verify_permissions and rpm_verify_ownership

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:
There can be cases in which a Profile requires that a file permission be
more strict than package default permissions, or a file be owned by root.
In these cases this rule will report the file changed by the Profile itself as a finding.

#### Rationale:
Not all permission or ownership requirements make sense to be incorporated by the
package, and currently there is no mechanism to waive these findings.

### CODE REVIEW

File: linux_os/guide/system/software/integrity/software-integrity/rpm_verification/rpm_verify_permissions/rule.yml [hunk 1]
```diff
@@ -67,8 +67,10 @@ ocil: |-
     is expected by the RPM database:
     <pre>$ rpm -Va | awk '{ if (substr($0,2,1)=="M") print $NF }'</pre>
 
-{{% if product == "rhel6" %}}
 warnings:
+    - general: |-
+        Profiles may require that specific files have stricter file permissions than defined by the vendor. Such files will be reported as a finding and need to be evaluated according to your policy and deployment environment.
+{{% if product == "rhel6" %}}
```
Comment (redhatrises) [inline linux_os/guide/system/software/integrity/software-integrity/rpm_verification/rpm_verify_permissions/rule.yml:73]: This doesn't appear to be RHEL6 specific, and there appears to be a problem for RHEL7, possibly RHEL8. Should we remove this?

Comment (matejak) [reply linux_os/guide/system/software/integrity/software-integrity/rpm_verification/rpm_verify_permissions/rule.yml:73]: The referenced BZ is filed against RHEL6. Are you sure about that?

**Comment by redhatrises** [comment]
There's #5712 opened against RHEL7. We should update the message while we are here if there are multiple problems against gdm in RHEL.

**Comment by matejak** [comment]
I would love to have something similar for `rpm_verify_ownership`, as ownership by `root` can be our stricter requirement than ownership defined by the RPM package.

File: linux_os/guide/system/software/integrity/software-integrity/rpm_verification/rpm_verify_permissions/rule.yml (hunk 1 above)
Comment (yuumasato) [reply linux_os/guide/system/software/integrity/software-integrity/rpm_verification/rpm_verify_permissions/rule.yml:73]: @redhatrises You mean to make the warning applicable to all products, right?

Comment (redhatrises) [reply linux_os/guide/system/software/integrity/software-integrity/rpm_verification/rpm_verify_permissions/rule.yml:73]: @yuumasato yeah

Comment (yuumasato) [reply linux_os/guide/system/software/integrity/software-integrity/rpm_verification/rpm_verify_permissions/rule.yml:73]: Hmm, the referenced BZ is a clone of RHEL7 bug, and the RHEL7 bug was solved: https://bugzilla.redhat.com/show_bug.cgi?id=1275532

I would still keep the warning specific for RHEL6.

**Comment by yuumasato** [comment]
I have added similar warning to `rpm_verify_ownership` rule.

**Comment by matejak** [comment]
I am merging the PR. If more warnings with greater specificity are desired, they can be discussed in subsequent PRs, but this one already has a significant positive impact.

File: linux_os/guide/system/software/integrity/software-integrity/rpm_verification/rpm_verify_permissions/rule.yml (hunk 1 above)
Comment (redhatrises) [reply linux_os/guide/system/software/integrity/software-integrity/rpm_verification/rpm_verify_permissions/rule.yml:76]: Ahhh... okay. Got it thanks for that.

**Pull request merged**